dev/gather_coverage.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# This file was modified from GAP-system's dev/ci-gather-coverage.sh.  Visit
+# their website at <https://www.gap-system.org/>.
+
+# NOTE: This needs to be executed from the FLINT directory *after* checks has
+# been made.
+
+set -ex
+
+find build -type f -name '*.gcno' -exec gcov -pb {} +
```

Comment:
I think no gcov files will be produced unless you also pass suitable linker flags (e.g. --coverage)

Reply:
Alright!